src/routes/getStatus.ts

```diff
@@ -3,14 +3,19 @@ import {Logger} from "../utils/logger";
 import {Request, Response} from "express";
 
 export async function getStatus(req: Request, res: Response): Promise<Response> {
+    const startTime = Date.now();
+    let value = req.params.value as string[] | string;
+    value = Array.isArray(value) ? value[0] : value;
```

Comment:
Why dos this accept an array if it only uses the first element?

Reply:
parameter can be repeated and it returns an array, see category

Reply:
see #235